pkg/connector/client.go

```diff
@@ -58,6 +62,20 @@ func NewTwitterClient(login *bridgev2.UserLogin, connector *TwitterConnector, cl
 		userCache: make(map[string]types.User),
 	}
 	client.SetEventHandler(tc.HandleTwitterEvent)
+	tc.matrixParser = &format.HTMLParser{
```

Comment:
I should probably make these fields optional at some point

```suggestion
	tc.matrixParser = &format.HTMLParser{
		TabsToSpaces:   4,
		Newline:        "\n",
		HorizontalLine: "\n---\n",
```